openmc/universe.py

```diff
@@ -118,6 +128,12 @@ def name(self, name):
         else:
             self._name = ''
 
+    @volume.setter
+    def volume(self, volume):
+        if volume is not None:
+            cv.check_type('universe volume', volume, Real)
```

Comment:
Perhaps- I don't have a strong opinion on the matter. I was just following the other setters for the universe class. Probably worth revisiting this for the API as a whole at some point.